tests/translator/test_managed_policies_translator.py

```diff
@@ -7,6 +7,22 @@ def create_page(policies):
         'Policies': map(lambda x: {'PolicyName': x[0], 'Arn': x[1]}, policies)
     }
 
+
+def test_load_local():
+    paginator = MagicMock()
+    iam = MagicMock()
+    iam.get_paginator.return_value = paginator
+
+    actual = ManagedPolicyLoader(iam).load()
+
+    assert not iam.get_paginator.called
+    assert isinstance(actual, dict)
+    assert actual.get('AmazonMechanicalTurkCrowdFullAccess')
+
+    # NOTE: this is a moving target that must be updated when ManagedPolicyLoader._local_managed_policies_map is updated
+    assert len(actual) == 401
```

Comment:
This test doesn't really tell us anything. If we want a test around the list of managed policies we should actually download the list of managed policies from IAM and do a shallow (or maybe even deep...) comparison to our list. The downside there is that we'll be forced to update our list every time a modification to the list of managed policy arises.

Reply:
It tells us that at least one of the expected policies is there, and that the number of entries is as expected. I think this is reasonable.